static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -144,21 +147,26 @@ function IssueContext(props: IssueContextProps) {
     }
   );
 
+  const {
+    isLoading: eventLoading,
+    isError: eventError,
+    data: event,
+  } = useQuery<Event>([`/issues/${props.dataRow['issue.id']}/events/oldest/`], {
+    staleTime: fiveMinutesInMs,
+  });
```

Comment:
Can we leave a comment here about how this oldest event is related to rendering the suspect commit?

Reply:
Yeah we should add a comment for sure.